packages/esm-patient-programs-app/src/programs/programs-form.component.tsx

```diff
@@ -146,152 +160,149 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({ closeWorkspace, patientUuid
         sub.unsubscribe();
       };
     },
-    [
-      selectedProgram,
-      patientUuid,
-      enrollmentDate,
-      completionDate,
-      userLocation,
-      currentEnrollment,
-      mutateEnrollments,
-      closeWorkspace,
-      t,
-    ],
+    [patientUuid, currentEnrollment, mutateEnrollments, closeWorkspace, t],
   );
+
+  const onError = (error) => {
+    console.error(error);
```

Comment:
why are you console logging the error, can we do something else with this function or just get rid of it if we don't need it.

Reply:
reslolved